DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/GroupDelegates/SelectDelegate.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.GroupDelegates
+@using Microsoft.Extensions.Configuration
+@model AddGroupDelegateViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/shared/cardWithButtons.css")" asp-append-version="true">
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/allDelegates.css")" asp-append-version="true">

```

Comment:
Can we have a single css import, and bring these together in an scss file.

Reply:
Yep, done.